packages/cashier/src/stores/__tests__/general-store.spec.ts

```diff
@@ -262,47 +248,20 @@ describe('GeneralStore', () => {
     });
 
     it('should perform proper init invocation when is_logged_in is equal to true', async () => {
-        const spyGetAdvertizerError = jest.spyOn(general_store, 'getAdvertizerError');
         const spyCheckP2pStatus = jest.spyOn(general_store, 'checkP2pStatus');
         general_store.root_store.client.is_logged_in = true;
         general_store.init();
 
         await waitFor(() => {
-            expect(spyGetAdvertizerError).toHaveBeenCalledTimes(1);
+            expect(spyCheckP2pStatus).toHaveBeenCalledTimes(1);
         });
-        expect(spyCheckP2pStatus).toHaveBeenCalledTimes(1);
+
+        // Don't remove eslint here as WS.wait is expected to be called 3 times from init and checkP2pStatus
         // eslint-disable-next-line testing-library/await-async-utils
-        expect(general_store.WS.wait).toHaveBeenCalledTimes(1);
+        expect(general_store.WS.wait).toHaveBeenCalledTimes(3);
         expect(general_store.root_store.modules.cashier.withdraw.check10kLimit).toHaveBeenCalledTimes(1);
     });
 
-    it('should set advertiser error', async () => {
-        const spySetP2pAdvertiserError = jest.spyOn(general_store, 'setP2pAdvertiserError');
-        await general_store.getAdvertizerError();
-
-        expect(spySetP2pAdvertiserError).toHaveBeenCalledWith('advertiser_error');
-    });
-
-    it('should set p2p advertiser error', () => {
-        general_store.setP2pAdvertiserError('p2p_advertiser_error');
-
-        expect(general_store.p2p_advertiser_error).toBe('p2p_advertiser_error');
-    });
-
-    it('should set is_p2p_visible equal to false, if there is a "RestrictedCountry" p2p advertiser error', () => {
-        general_store.setP2pAdvertiserError('RestrictedCountry');
-        general_store.checkP2pStatus();
-
-        expect(general_store.is_p2p_visible).toBeFalsy();
-    });
-
-    it('should set is_p2p_visible equal to false, if there is a "RestrictedCurrency" p2p advertiser error', () => {
-        general_store.setP2pAdvertiserError('RestrictedCurrency');
-        general_store.checkP2pStatus();
-
-        expect(general_store.is_p2p_visible).toBeFalsy();
-    });
-
```

Comment:
Why are we removing the test cases? 😄

Reply:
this one was done by carol, we removed these because initially we were checking if there was an error from `p2p_advertiser_info` in order to determine if p2p should be shown in the onboarding in cashier

Reply:
so we removed all these test cases since we don't check for p2p advertiser error anymore